### PR TITLE
Sections endpoint

### PR DESCRIPTION
**Pull request by gislenius**

### CODE REVIEW

**Comment by gislenius** [comment]
TODO:

- Swagger documentation
- main documentation

**Comment by gislenius** [comment]
Just one remaining failing test that should be straightforward to fix. @dknochen you can already review! :-)

File: documentation/4-sections-0-sections.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+In a fiscal filing, for example, there is very often a section for a balance sheet,
+for an income statement, etc.
+
+Sections contain the metadata (report elements) used and shared by all its components.
+
+A section may only have one component (which happens quite often), or several.
+
+Sections are identified with a URI.
+
+Spreadsheet analogy
```
Comment (wcandillon) [inline documentation/4-sections-0-sections.md]: @gislenius maybe remove this section.

**Comment by wcandillon** [comment]
:+1:

**Pull request merged**